### ISSUE TITLE
Element Modifiers in components

### ISSUE THREAD

**Issue by josemarluedke** [ember-template-lint/ember-template-lint#696]
I'm using Ember canary (`27255073091ef4c020ed8bf552c54f5b3f54d25b`) where it has forwarding of element modifiers enabled (`EMBER_GLIMMER_FORWARD_MODIFIERS_WITH_SPLATTRIBUTES`). 

I have a component that I'm passing a modifier to it, while the application works as expected, template lint has a compile error.

```
  -:-  error  Compile Error: Element modifiers are not allowed in components  undefined
```

I could not find any rules for that, so my understanding is that we need to upgrade glimmer compiler here?

**Comment by larryh** [comment]
Hi,

Was this fixed? Asking because this code in a component template spits out the following error:

`{{on "click" (fn this.toggleAccordions "our-site" "problem")}}`

`Compile Error: Element modifiers are not allowed in components  undefined`

But the code works fine.

**Comment by rwjblue** [comment]
Yep, it was fixed in #709. An update to your `ember-template-lint` version should bring in those (and other) fixes.

**Comment by larryh** [comment]
Got it. We just discussed this in Ember Discuss. Thanks!
